source/stdx/allocator/mallocator.d

```diff
@@ -201,6 +201,9 @@ version (Windows)
 /**
    Aligned allocator using OS-specific primitives, under a uniform API.
  */
+version (D_BetterC) {} else version = HasMemAlign;
```

Comment:
I think this needs to be `version (WebAssembly) {} else version = HasMemAlign;` since memalign is actually available in betterC.